### PR TITLE
Create draft of singularity-htc.md

### PR DESCRIPTION
**Pull request by rachellombardi**
Needs review once `apptainer` is installed on HTC system

### CODE REVIEW

File: uw-research-computing/singularity-htc.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+## 1. Choose or create an Apptainer Image
+To run an Apptainer job, it is necessary to first choose a preexisting Apptainer image or create your own. 
+
+### A. Preexisting Images
+Like Docker, it is easiest to use Apptainer in your jobs by finding a preexisting image that contains your desired software and environment setup. 
+
+Preexisting "base" images can be found in the Apptainer library or from reputable sources such as the [OSG](https://portal.osg-htc.org/documentation/htc_workloads/using_software/available-containers-list/). sAdditionally, it to convert a Docker container (e.g your own or from DockerHub) to a Apptainer image.
```
Comment (rachellombardi) [inline uw-research-computing/singularity-htc.md:18]:
```suggestion
Preexisting "base" images can be found in the Apptainer library or from reputable sources such as the [OSG](https://portal.osg-htc.org/documentation/htc_workloads/using_software/available-containers-list/). Additionally, it to convert a Docker container (e.g your own or from DockerHub) to a Apptainer image. 
```

File: uw-research-computing/singularity-htc.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+## 7. Use an Apptainer Container in HTC Jobs
+
+HTCondor will automatically fetch and  prepare your Apptainer environment for your high-throughput jobs as long as we provide HTCondor with the `.sif` file information in your job's submit file. Specifically, we need to provide HTCondor with the name of the `container_image` and to tell HTCondor where to find the `.sif` file to transfer to the job using `transfer_input_files` like so: 
+
+```
+# HTC Submit File
+
+# Provide HTCondor with the name of your .sif file and universe information
+container_image = my-container.sif
+universe = container
```
Comment (rachellombardi) [inline uw-research-computing/singularity-htc.md:151]: Write a comment next to this that it is optional

Comment (ChristinaLK) [reply uw-research-computing/singularity-htc.md:151]: `universe = container` is optional?

File: uw-research-computing/singularity-htc.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+# 6. Exit your HTCondor Interactive Job
+
+Now that we have prepared our Apptainer image and tested it, we are ready to exit our interactive session. When we do, our build file will return to our `/home` directory on the submit server.  To exit, run:
+
+```
+[alice@build]$ exit
+```
+
+## 7. Use an Apptainer Container in HTC Jobs
```
Comment (ChristinaLK) [inline uw-research-computing/singularity-htc.md:142]: I would actually make this first in the guide. I'll submit a quick PR against the Docker guide to show what I mean.

**Review by ChristinaLK — COMMENTED** [review]
Started reviewing - but it feels like there some things to still edit/test before a real review?

**Comment by rachellombardi** [comment]
@ChristinaLK You are correct, this guide is not ready to be merged. Andrew is going to test the commands now that apptainer is installed, and add some of additional workflow ideas.

**Pull request closed without merging**